### DISCUSSION TITLE
className popoverProps on MultiSelect?

### DISCUSSION THREAD

**Discussion by stoplion** [palantir/blueprint#5880] [Q&A] [answered]
#### Environment

OS X 10.14.6
Blueprint 3.23.1

#### Question

I'm trying to pass popoverProps to MultiSelect, to style the popover but it seems that the className is not passed to the popover. Is this expected?

**Comment by mormahr** [comment]
Looking at the source code it seems like it should pass them. The `className` set via `popoverProps` will be concatenated to the internal classes.

https://github.com/palantir/blueprint/blob/fddba2c6472d2a860288294e3f29c2319d5ba64a/packages/select/src/components/select/multiSelect.tsx#L139

Can you provide a reproduction case (e.g. code sandbox)?

**Comment by adidahiya** [comment, marked as answer]
@stoplion you may be looking for `popoverProps.popoverClassName`
